assemble/src/main/scripts/create-jshell.sh

```diff
@@ -40,19 +40,29 @@ function addClientBuild() {
 cat <<EOF
   URL clientPropUrl = 
     AccumuloClient.class.getClassLoader().getResource("accumulo-client.properties");
-  String accumuloProp; AccumuloClient client = null;
-  
+  AccumuloClient client = null;
+      
   // Does Accumulo properties exists?
   if (clientPropUrl != null) {
   
-    // Build Accumulo Client
-    accumuloProp = clientPropUrl.getFile();
-    System.out.println("Building Accumulo client using properties file below: \n" + accumuloProp);
-    client = Accumulo.newClient().from(accumuloProp).build();
-    System.out.println("Use " + "\"client\"" + " to interact with Accumulo \n");
-  } else {
-      System.out.println("accumulo-client.properties was not found on the classpath \n");
-  }
+    // Load in Accumulo properties
+    try (InputStream in = clientPropUrl.openStream()) {
+      Properties prop = new Properties(); 
```

Comment:
+1 to this strategy. I briefly looked into adding a method that does this directly to AccumuloClient.Builder, but it may not be necessary, since a user can do it themselves. It's also possible we could create an AccumuloClient.Builder where it just does `fromClasspath()` and auto-detects the presence of a client properties file, if it exists. But either way, that would be a separate effort, and this strategy should work fine here for now.

One thing you could do here is limit the scope of the try-with-resources block by declaring the `prop` outside the block, and only doing the load call inside the block. Something like:

```java
  var prop = new Properties();
  try (var in = clientPropUrl.openStream()) {
    prop.load(in);
  }
  // create client with the prop variable after the try-with-resources block is closed

Reply:
Sounds good! I can work on creating the fromClasspath() method on a separate PR.